### PR TITLE
Fix build on Windows in CI

### LINKED ISSUE #207: Fix windows build or disable http client for Windows

### PR DESCRIPTION
**Pull request by petertrr**
### What's done:
* Update build_and_test.yml: gradle-cache-action ignores properties without value, so added explicit `disableRedundantTargets=true`
* Added more logging
* Always close ktor client

Closes #207

### CODE REVIEW

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -72,32 +72,26 @@ jobs:
...
-      - name: Install curl Windows
-        continue-on-error: true
-        run: pacman -S --noconfirm mingw-w64-x86_64-curl
-        shell: msys2 {0}
-
       - name: Cache konan
         uses: actions/cache@v2
         with:
-          path: ~/.konan
+          path: 'C:\Users\runneradmin\.konan'
```
Comment (kgevorkyan) [inline .github/workflows/build_and_test.yml:94]: There is no any env variable for this?

File: save-core/src/jvmTest/kotlin/org/cqfn/save/core/test/utils/TestUtilsJvm.kt [hunk 2]
```diff
@@ -43,10 +43,11 @@ actual suspend fun downloadFile(url: String, fileName: String): String {
     if (!file.exists()) {
         val httpResponse: HttpResponse = client.get(url)
         val responseBody: ByteArray = httpResponse.receive()
+        println("Writing ${responseBody.size} bytes into ${file.path}")
```
Comment (kgevorkyan) [inline save-core/src/jvmTest/kotlin/org/cqfn/save/core/test/utils/TestUtilsJvm.kt:46]: May be `logDebug`?

File: .github/workflows/build_and_test.yml (hunk 1 above)
Comment (petertrr) [reply .github/workflows/build_and_test.yml:94]: I tried `HOME`, `HOMEPATH` and `USERPROFILE`, they don't seem to work, at least when configuring a step

File: save-core/src/jvmTest/kotlin/org/cqfn/save/core/test/utils/TestUtilsJvm.kt (hunk 2 above)
Comment (petertrr) [reply save-core/src/jvmTest/kotlin/org/cqfn/save/core/test/utils/TestUtilsJvm.kt:46]: Yes, changed

**Review by kgevorkyan — APPROVED** [review]

**Pull request merged**